models/Staticroute.php

```diff
@@ -394,13 +394,17 @@ public function getReverse()
     }
 
     /**
-     * @param string|array $siteId
+     * @param string|array|null $siteId
      *
      * @return $this
      */
     public function setSiteId($siteId)
     {
         $result = [];
+        
+        if (null === $siteId) {
+            return;
```

Comment:
I think this could stay. Null should set an empty array.
```suggestion
            $siteId = [];
```